resources/040_deployment.yaml.tmpl

```diff
@@ -18,23 +15,33 @@ spec:
       labels:
         name: stuck-ebs-vols
     spec:
+      initContainers:
+      - name: setupcreds
+        image: quay.io/lseelye/yq-kubectl:$INIT_IMAGE_VERSION
```

Comment:
Need this image in a redhat or openshift repository, not a personal repo.  Need CI for building, maybe can leverage a build trigger on quay.io?

Reply:
@jewzaam Agreed. Do we have any docs/guidance on how to migrate this over to an openshift git org repo and then to the red hat quay account with triggers/etc?